/**:
  ros__parameters:
    # --- 基本設定 ---
    speed_scale:          1.0
    steer_scale:          1.0
    timer_hz:             30.0

    # --- 軸設定 ---
    speed_axis_index:    1  # 左スティック上下で速度制御 humble: 1
    steer_axis_index:    2  # 右スティック左右でステア制御 humble: 2

    # --- モード切替ボタン ---
    joy_button_index:     3   # △ボタンで手動モード humble: 3, foxy: 2
    ack_button_index:     0   # □ボタンで自律モード humble: 0, foxy: 3

    # --- start/stop トリガボタン ---
    start_button_index:   7   # R2 ボタンで start=true humble: 7, foxy: 7
    stop_button_index:    6   # L2 ボタンで start=false humble: 6, foxy: 6

## button index (DualShock4) humble
## humble humble humble
# 0: □
# 1: ×
# 2: ○
# 3: △
# 4: L1
# 5: R1
# 6: L2
# 7: R2
# 8: share
# 9: option
# 10: home
# 11: L3
# 12: R3

## axes index (DualShock4) for D-pad（オフセット調整用）
# →: axes[6] == -1.0  → steer_offset +
# ←: axes[6] ==  1.0  → steer_offset –
# ↑: axes[7] ==  1.0  → speed_offset +
# ↓: axes[7] == -1.0  → speed_offset –